dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/ArticleRepositoryImpl.java

```diff
@@ -37,8 +37,8 @@ public long count() {
   @Override
   public PageResponse<Article> find(PageRequest pageRequest) {
     StringBuilder query =
-        new StringBuilder("SELECT " + IDENTIFIABLE_COLUMNS + ", text").append(" FROM articles");
-
+        new StringBuilder(
```

Comment:
Braucht man hier diesen `StringBuilder`?

Reply:
ja, da der builder in methoden weitergereicht wird und dort fragemente appended werden. rückgabewert der methoden ist void. würde man einen string reinreichen, müsste man alle methoden umändern, dass sie den erweiterten string zurückgeben und dann nach dem aufrufen auswerten. ergo: builder lassen, da weniger code und einfacher handhabbar